src/App.jsx

```diff
@@ -14,45 +16,106 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_BY_ALPHABET = 'name';
+const SORT_BY_LENGTH = 'length';
+const REVERSE = 'reverse';
+
+export function getPreparedGoods(goods, howSort, reverse) {
+  const preparedGoods = [...goods];
+
+  if (howSort) {
+    preparedGoods.sort((good1, good2) => {
+      switch (howSort) {
+        case SORT_BY_ALPHABET:
+          return good1.localeCompare(good2);
+
+        case SORT_BY_LENGTH:
+          return good1.length - good2.length;
+
+        default:
+          return 0;
+      }
+    });
+  }
+
+  if (reverse) {
+    preparedGoods.reverse();
+  }
+
+  return preparedGoods;
+}
 
-    <ul>
-      <li data-cy="Good">Dumplings</li>
-      <li data-cy="Good">Carrot</li>
-      <li data-cy="Good">Eggs</li>
-      <li data-cy="Good">Ice cream</li>
-      <li data-cy="Good">Apple</li>
-      <li data-cy="Good">...</li>
-    </ul>
-  </div>
-);
+export const App = () => {
+  const [sortBy, setSortBy] = useState(null);
+  const [reverse, setReverse] = useState(null);
+  const visibleGoods = getPreparedGoods(goodsFromServer, sortBy, reverse);
+
+  return (
+    <div className="section content">
+      <div className="buttons">
+        <button
+          type="button"
+          className={
+            cn(
+              'button',
+              'iso-inf',
+              { 'is-light': sortBy !== SORT_BY_ALPHABET },
```

Comment:
check your demo link , when I click on this button it shoud be blue
![image](https://github.com/mate-academy/react_list-of-goods-js/assets/91946036/685976c1-a878-488b-aa81-ef793fa89859)

check your classes

I think it should be 'button is-info',